index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+</head>
+
+<body>
+  <script src="./implement-clearAllTimeout.js"></script>
```

Comment:
setTimeout in window and non-window environments
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/globalThis